config/locales/en.yml

```diff
@@ -623,8 +623,10 @@ en:
       message_placeholder: "Enter your message..."
   static_pages:
     donate:
+      give: "Give"
+      give_sentence: "them a chance to be loved"
       contact_us: "Contact us"
-      contact_us_to_donate: "%{href} to ask how you can donate"
+      contact_us_to_donate: "to ask how you can donate"
```

Comment:
We can remove this per my comment above on the button.

Reply:
done @kasugaijin 

- [x] SS Attatched
![image](https://github.com/user-attachments/assets/eb0dc0bd-a3c3-4a14-8fb3-08dab6523420)